src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+const CalcBody = () => (
+  <section>
+    <div className="container">
+      <div className="button">0</div>
+      <div className="calculator-left">
+        <div className="button">AC</div>
+        <div className="button">+/-</div>
+        <div className="button">%</div>
+        <div className="button">7</div>
+        <div className="button">8</div>
+        <div className="button">9</div>
+        <div className="button">4</div>
+        <div className="button">5</div>
+        <div className="button">6</div>
+        <div className="button">1</div>
+        <div className="button">2</div>
+        <div className="button">3</div>
+        <div className="button">0</div>
+        <div className="button">.</div>
+      </div>
+      <div className="calculator-right">
+        <div className="operator">+</div>
+        <div className="operator">x</div>
+        <div className="operator">-</div>
+        <div className="operator">+</div>
+        <div className="operator">=</div>
+      </div>
+    </div>
+  </section>
+);
+
+const Calculator = () => (
+  <div>
+    <CalcBody />
+  </div>
+);
+
```

Comment:
- [ ] According to the project requirement your app should match the following design. Please implement it.
### Your design
![his](https://github.com/Katended/math-magicians/assets/81727988/426be167-fe18-4d85-b3b5-217ef2c71ff8)

### Original design
![original](https://github.com/Katended/math-magicians/assets/81727988/ef090be5-68a3-4315-a97b-0b6bb9262169)